### PR TITLE
fix: docker compose volumes

### PR DESCRIPTION
**Pull request by Dmitrii-Shevchenko**
EPMLSTRCMW-170
Create docker compose file for production start

### CODE REVIEW

**Review by OstapStrashevskii — APPROVED** [review]
Looking good =)

**Review by GrigoriiBerezin — APPROVED** [review]
Looks pretty good. Thank you!

File: docker-compose.yml [hunk 1]
```diff
@@ -1,5 +1,31 @@
...
+      ./docker/postgres:/var/lib/postgresql/data
+    restart: always
+
+  mongo:
+    image: "mongo"
+    container_name: "mongo"
+    environment:
+      MONGO_INITDB_ROOT_USERNAME: mongoUser
+      MONGO_INITDB_ROOT_PASSWORD: password
+      MONGO_INITDB_DATABASE: mongo
```
Comment (iliaonishchenko) [inline docker-compose.yml]: Not sure if it's a good idea to leave usernames and passwords in plaintext.

**Review by iliaonishchenko — COMMENTED** [review]
?

File: docker-compose.yml (hunk 1 above)
Comment (Dmitrii-Shevchenko) [reply docker-compose.yml]: Of course! Already fixed

**Review by IaroslavTavchenkov — APPROVED** [review]
It seems nice! Good job! :)

**Comment by iliaonishchenko** [comment]
Now you need to squash commits and I'll approve it.

**Review by mnesvit — APPROVED** [review]

**Review by AndreiMaksimenko — APPROVED** [review]

**Review by iliaonishchenko — APPROVED** [review]

**Pull request merged**